src/bika/lims/browser/contact.py

```diff
@@ -78,14 +78,26 @@ def get_users(self):
         users_view = UsersOverviewControlPanel(self.context, self.request)
         return users_view.doSearch("")
 
+    @view.memoize
+    def get_clients_groups(self):
+        """Returns the client-specific groups
+        """
+        groups = []
+        uc = api.get_tool("uid_catalog")
+        for brain in uc(portal_type="Client"):
+            client = api.get_object(brain)
+            groups.append(client.group_id)
```

Comment:
Maybe we should consider the group id as metadata column?

Reply:
Done with https://github.com/senaite/senaite.core/pull/2527/commits/5d205865cc11240517b0915826865e1b413c9183 and https://github.com/senaite/senaite.core/pull/2527/commits/add880883fdde48952e67cbd918f98abd6226f16